clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -734,6 +736,14 @@ private static void estimateDataRecoveryTime(CommandLine cmd) {
     }
   }
 
+  private static void deleteValueSchema(CommandLine cmd) {
+    String storeName = getRequiredArgument(cmd, Arg.STORE);
```

Comment:
So if we implement the auto schema cleanup logic, why do we still need the manual cleanup?
Manual cleanup can be unsafe without proper check.